docs/Data_Submission_Portal/Users_Guide/Troubleshooting.md

```diff
@@ -0,0 +1,1236 @@
+            "submitter_id": "Alignment_Workflow_000088"
+        },
+        "type": "aligned_reads"
+    }
+]
+```
+```Response2
+{
+  "cases_related_to_created_entities_count": 1,
+  "cases_related_to_updated_entities_count": 0,
+  "code": 200,
+  "created_entity_count": 1,
+  "entities":
+    [
+        {
+            "action":"create",
+            "errors":[],
+            "id":
+                [
+                    {
+                        "id":"ad19702e-7951-47a8-b547-00b077b1547a"
+                    }
+                ],
+            "related_cases":[
+                {
+                    "id":"221db987-6008-4839-a735-63869761b4f9","submitter_id":"GDC-INTERNAL-000088"
+                }
+            ],
+            "type":"aligned_reads",
+            "unique_keys":
+                [
+                    {
+                        "project_id":"GDC-INTERNAL",
+                        "submitter_id":"Aligned_Reads_000089_v2"
+                    }
+                ],
+            "valid":true,
+            "warnings":[]
+        }
+    ],
+    "entity_error_count":0,
+    "message":"Transaction successful.",
+    "success":true,
+    "transaction_id":6373005,
+    "transactional_error_count":0,
+    "transactional_errors":[],
+    "updated_entity_count":1
+}
+```
+
+```Request1
+{
+  "projects": {
+    "code": "INTERNAL"
+  }, 
+  "disease_type": "Adenomas and Adenocarcinomas", 
+  "primary_site": "Bladder",
+  "project_id": "GDC-INTERNAL", 
+  "type": "case",
+  "submitter_id": "GDC-INTERNAL-000101"
+}
+```
+```Response1
+{
+  "cases_related_to_created_entities_count": 0,
+  "cases_related_to_updated_entities_count": 0,
+  "code": 400,
+  "created_entity_count": 0,
+  "entities":
+    [
+        {
+            "action":"create",
+            "errors":
+                [
+                    {
+                        "keys":["submitter_id"],
+                        "message":"Unable to validate case against dbGaP. [500] - Internal server error: Unable to cross reference cases with dbGaP. Either this project is not registered in dbGaP or we were temporarily unable to communicate with dbGaP. Please try again later.",
+                        "type":"NOT_FOUND"
+                    }
+                ]
+            "id":
+                [
+                    {
+                        "id":"GDC-INTERNAL-000101"
+                    }
+                ],
+            "related_cases":[],
+            "type":"case",
+            "unique_keys":[
+                {
+                    "project_id":"GDC-INTERNAL"
+                }
+            ],
+            "valid":false,
+            "warnings":[]
+        }
+    ],
+    "entity_error_count":1,
+    "message":"Transaction aborted due to 1 invalid entity.",
+    "success":false,
+    "transaction_id":6373005,
+    "transactional_error_count":0,
+    "transactional_errors":[],
+    "updated_entity_count":0
+}
+```
+
+#### UNALLOWED_GENCODE_VERSION Messages
```

Comment:
You can remove this section, external submitters will never provide gencode versions to the GDC.